core/src/main/java/discord4j/core/shard/GatewayBootstrap.java

```diff
@@ -163,8 +167,8 @@
      * {@link GatewayOptions} object passed to the {@link GatewayClient} factory in connect methods.
      *
      * @param optionsModifier {@link Function} to transform the {@link GatewayOptions} type to provide custom
-     * {@link GatewayClient} implementations a proper configuration object.
-     * @param <O2> new type for the options
+     *                        {@link GatewayClient} implementations a proper configuration object.
```

Comment:
To avoid these you should apply our [code style](https://github.com/Discord4J/meta/blob/master/Discord4J.xml) to the project.